### PR TITLE
tests: Test translations

### PR DESCRIPTION
**Pull request by marusak**
Validate that translations do not break pages and also check for used
locale and that it is valid.

Tested with both breaking translations, just like [here](https://github.com/cockpit-project/cockpit/issues/10604#issuecomment-443625840) and with breaking locale like [here](https://github.com/cockpit-project/cockpit/commit/a395a4a5155e62b24cdb8afa3697618c31282e1f)

### CODE REVIEW

**Comment by marusak** [comment]
A few issues:

1. atomics don't have storage page (I will just drop this page)
2. distropkg does not have the new id I introduced (I need to write a different selector)
3. Ubuntus/Debians don't recognize `ca-es` as a valid locale

File: test/verify/check-pages [hunk 1]
```diff
@@ -216,6 +216,62 @@ OnCalendar=daily
...
         b.wait_text("#password-group > label", "Passwort")
 
+        # Test all languages
+        # Test that pages do not oops and that locale is valid
+
+        def line_sel(i):
+            if m.image == "rhel-7-6-distropkg":
+                return '.terminal div:nth-child(%d)' % i
+            else:
+                return '.terminal .xterm-accessibility-tree div:nth-child(%d)' % i
```
Comment (martinpitt) [inline test/verify/check-pages]: FWIW,  I think it's fine  to just skip that test on *-distropkg, atomics, and others -- testing this on one release (like fedora) seems fine. Feel free to make it a new test case, then it's easier to do so.

Comment (marusak) [reply test/verify/check-pages]: I am going to tests this only on F29 and F30. I don't want to create a separate test case as booting machine takes some time and this is rather slow anyway.
Pushed a edited version.

**Comment by marusak** [comment]
Hmm


```
Process 7183 (cockpit-pcp) of user 1000 dumped core.
/usr/libexec/cockpit-pcp: bridge was killed: 11
Wrote JS log to TestPages-testBasic-fedora-30-127.0.0.2-2701-FAIL.js.log
Journal extracted to TestPages-testBasic-fedora-30-127.0.0.2-2701-FAIL.log
Wrote screenshot to TestPages-testBasic-fedora-30-127.0.0.2-2701-FAIL.png
Wrote HTML dump to TestPages-testBasic-fedora-30-127.0.0.2-2701-FAIL.html
Wrote JS log to TestPages-testBasic-fedora-30-127.0.0.2-2701-FAIL.js.log
Journal extracted to TestPages-testBasic-fedora-30-127.0.0.2-2701-FAIL.log
Traceback (most recent call last):
  File "/build/cockpit/test/common/testlib.py", line 776, in tearDown
    self.check_journal_messages()
  File "/build/cockpit/test/common/testlib.py", line 958, in check_journal_messages
    raise Error(first)
testlib.Error: Process 7183 (cockpit-pcp) of user 1000 dumped core.
```

**Comment by martinpitt** [comment]
@marusak: That's good old known issue #6108 again. It seems  these naughty overrides weren't copied for fedora-30, but should be.

**Comment by marusak** [comment]
> @marusak: That's good old known issue #6108 again. It seems these naughty overrides weren't copied for fedora-30, but should be.

Ah great, I just started looking into this. But it seems that #6108 was not happening on F30 (that's why it isn't copied) but now with this test it is happening every single time. I will push here another commit with this naughty for F30.

**Comment by martinpitt** [comment]
There is [another SELinux violation](https://209.132.184.41:8493/logs/pull-11565-20190410-081919-60287268-verify-fedora-29/log.html#266) in the new test.

**Comment by marusak** [comment]
> There is another SELinux violation in the new test.

I was not able to reproduce this locally so I triggered the test and it passed. So I assume it was just a flake.

**Comment by martinpitt** [comment]
New test succeeded everywhere. Thanks!

**Pull request merged**